.env.sample

```diff
@@ -20,7 +20,8 @@ TAIKO_L2_ADDRESS=0x0000777700000000000000000000000000000001
 
 # A L2 account private key for building throw-away L2 blocks, for more detailed information, please
 # see whitepaper's `5.5.1 Invalid Blocks`.
-L2_THROWAWAY_BLOCK_BUILDER_PRIVATE_KEY=92954368afd3caa1f3ce3ead0069c1af414054aefe1ef9aeacc1bf426222ce38 # LibAnchorSignature.K_GOLDEN_TOUCH_PRIVATEKEY
+L2_THROWAWAY_BLOCK_BUILDER_PRIVATE_KEY=92954368afd3caa1f3ce3ead0069c1af414054aefe1ef9aeacc1bf426222ce38
+# LibAnchorSignature.K_GOLDEN_TOUCH_PRIVATEKEY
```

Comment:
lets change this comment to be at the top of 
```suggestion
# LibAnchorSignature.K_GOLDEN_TOUCH_PRIVATEKEY
L2_THROWAWAY_BLOCK_BUILDER_PRIVATE_KEY=92954368afd3caa1f3ce3ead0069c1af414054aefe1ef9aeacc1bf426222ce38
```

lets change the comment to be at the top of the actual value?

Reply:
OK I've moved it. I'm not sure what the `K_GOLDEN_TOUCH_PRIVATEKEY` is and how that's related to the `L2_THROWAWAY_BLOCK_BUILDER_PRIVATE_KEY`.